logging/google/cloud/logging/logger.py

```diff
@@ -231,17 +248,23 @@ def log_struct(self, info, client=None, labels=None, insert_id=None,
         :param http_request: (optional) info about HTTP request associated with
                              the entry.
 
+        :type resource :class:``google.cloud.logging.resource.Resource`
```

Comment:
You have two backticks before `google` (also drop in a tilde `~` before `google`)